### PR TITLE
recovery shell: print zerrors since the last command was executed

### PR DESCRIPTION
**Pull request by zdykstra**
I'm not super happy with how `print_kmsg_logs` is shaking out, but there's only so much time I'm willing to spend chasing elegance.

This is modestly tested, and could likely use further refinement after people play with it.

### CODE REVIEW

File: zfsbootmenu/install-helpers.sh [hunk 1]
```diff
@@ -151,7 +151,7 @@ create_zbm_profiles() {
 	[ -f /etc/profile ] && source /etc/profile
 	[ -f /lib/zfsbootmenu-completions.sh ] && source /lib/zfsbootmenu-completions.sh
 
-	export PS1="\[\033[0;33m\]zfsbootmenu\[\033[0m\] \w > "
+	export PS1='\$( /libexec/recovery-error-printer )\[\033[0;33m\]zfsbootmenu\[\033[0m\] \w > '
```
Comment (ahesford) [inline zfsbootmenu/install-helpers.sh:154]:
```suggestion
	export PS1='\[\033[0;33m\]zfsbootmenu\[\033[0m\] \w > '
	export PROMPT_COMMAND="/libexec/recovery-error-printer"
```

Comment (zdykstra) [reply zfsbootmenu/install-helpers.sh:154]: You can tell I don't actually use Bash as an interactive shell. I didn't know `PROMPT_COMMAND` existed.

Comment (ahesford) [reply zfsbootmenu/install-helpers.sh:154]: That's fair. We all know bash is and forever will be a programming language first.

**Review by ahesford — APPROVED** [review]
This is a good idea. Printing these messages in red might be nice, but isn't necessary.

**Comment by zdykstra** [comment]
This is **good enough** for now. I'll think a bit more about how I want to refine `print_kmsg_logs` - but this simple solution is fine for now.

**Pull request merged**